### PR TITLE
fix(STK-267): strategies not working properly with new token picker

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Description
* Updates our existing strategies to fit the new frequency picker constrains
* Updates our frequency card picker so it can handle the strategy selection

## Visual Evidence
https://www.loom.com/share/5210e383aeb54a9891eada4b09d584fb?sid=7d350ff1-231d-41fa-8925-eb964ea87359

### CODE REVIEW

File: packages/app/components/stackbox/FrequencyOptionsCard.tsx [hunk 1]
```diff
@@ -2,13 +2,32 @@
...
+    case FREQUENCY_OPTIONS.hour:
+      return days * 24;
+    case FREQUENCY_OPTIONS.day:
+      return days;
+    case FREQUENCY_OPTIONS.week:
+      return daysToWeeks(days);
+    case FREQUENCY_OPTIONS.month:
+      return days / 30;
+    default:
+      throw new Error("Invalid frequency option");
```
Comment (berteotti) [inline packages/app/components/stackbox/FrequencyOptionsCard.tsx]: We are throwing an error which is never caught, maybe we should log a message and return something.

File: packages/app/components/strategies/constants.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ export const STRATEGY_CATEGORIES: { [chainId: number]: ChainStrategies } = {
         {
           id: 3,
           buyToken: gnosisTokens.WETH,
-          daysAmount: 10,
+          daysAmount: 4,
```
Comment (berteotti) [inline packages/app/components/strategies/constants.tsx:96]: I know that 10 days of hourly trades is a lot. Any specific reason for choosing 4?

Comment (ElRodrigote) [reply packages/app/components/strategies/constants.tsx:96]: It fits the current constraint of the maximum hours per hourly trade accepted in our custom timeframe input. Right now, we have a maximum of 96 hours, so 4 days * 24 hours = 96.

We can change any of these variables, but I made this change to fit the constraints explained above.

File: packages/app/components/stackbox/FrequencyOptionsCard.tsx (hunk 1 above)
Comment (ElRodrigote) [reply packages/app/components/stackbox/FrequencyOptionsCard.tsx]: You're right! I'll console the error and return either the max or min value allowed for the selected frequency.

**Review by berteotti — APPROVED** [review]
Looks good!

**Pull request merged**